src/WebJobs.Script/Binding/DocumentDBBinding.cs

```diff
@@ -102,15 +102,7 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes()
             }
             else if (Access == FileAccess.Write && _bindingDirection == BindingDirection.Out)
             {
-                IAsyncCollector<JObject> collector = await context.Binder.BindAsync<IAsyncCollector<JObject>>(runtimeContext);
```

Comment:
This is just the first in a set of planned refactorings I'll be doing to these bindings to winnow them down as slim as possible, so all the bindings share the same centralized code (reduce duplication as I've done here).